buildtools/errors.go

```diff
@@ -4,5 +4,6 @@ import "errors"
 
 // Analysis errors.
 var (
-	ErrNoRevisionForPackage = errors.New("no revision found for package name")
+	ErrNoRevisionForPackage   = errors.New("no revision found for package name")
+	ErrNoRevisionForPackageOk = errors.New("no revision found for package name")
```

Comment:
1. Avoid having the same error message for multiple errors; this makes it very difficult to debug a failure after the fact because it's hard to tell what error caused the failure.
2. This name is a bit misleading because it's too general. In general, you want to name for ergonomics, which sometimes means avoiding generality (I know this is a hard habit to get out of). In this case, I'd probably call it something like `ErrPackageIsIgnored`.